sites/docs/src/content/docs/tutorials/tests_and_test_data/nf-test_writing_tests.md

```diff
@@ -128,24 +136,44 @@ nextflow_process {
     tag "abricate"
     tag "abricate/summary"
 
+    setup {
+        run("ABRICATE_RUN") {
+            script "../../run/main.nf"
+            process {
+            """
+            input[0] = Channel.fromList([
+                            tuple([ id:'test1', single_end:false ], // meta map
```

Comment:
Do we still need to specify that this is the meta map ?
Also it could be nice to setup the `single_end` only if used.
```suggestion
                            tuple([ id:'test1' ],
```